TDP/tdp-51.md

```diff
@@ -0,0 +1,74 @@
+# TreasuryDAO Charter
+
+## Quick Summary
+This document should serve as a basic pre-draft for the upcoming codified TreasuryDAO charter for the next 12 months (from Governance Resolution of this Charter) under which the TreasuryDAO signers will operate and stand by.
+
+Every sector of the treasury has a particular goal in mind, and a Treasury Charter ensures we are accountable for, and execute on this prescribed goal setting in an open and transparent way.
+The document will contain four major sections, describing in detail the purpose, key activities, additional responsibilities, and relevant sub-committees in detail.
+It is imperative we bring forward a document that can stand the test of time in any kind of market condition, as well as serve as a bookmark of reference if anyone wishes to observe the legitimacy and correct execution of the operational functioning of the DAO in the wider mStable ecosystem.
+Comments and feedback is highly encouraged, and a highly fine-tuned charter will make the life of all participants easier, more enjoyable, and showcase the professionalism under which we steer the Treasury forward.
+
+## Abstract
+The following section will outline the purpose of the TreasuryDAO, as well as some guiding principles under which the signers base their decision-making process on when it comes to managing funds, securing runway, as well as ensuring the appropriate custodianship over the mStableDAO treasury side.
+
+It is the mission of the TreasuryDAO to secure runway to ensure that the mStableDAO Operations are sufficiently funded while investing a portion of its assets into key opportunities in the ecosystem if the budget permits.
```

Comment:
Please add a `motivation` part. Just outlining quickly why this is important. You can just reshift some of the passages around.